### ISSUE TITLE
Does the kubernetes-client 1.4.33 support the kubernetes1.5.1 or not?

### ISSUE THREAD

**Issue by zhaorendong** [fabric8io/fabric8#6706]
Does the kubernetes-client 1.4.33 support the kubernetes1.5.1 or not?

**Comment by jimmidyson** [comment]
Yes but it doesn't have coverage for all resource types. Are you having problems using it with 1.5.1?

**Comment by zhaorendong** [comment]
When I rollingupdate ReplicationController that used  kubernetes-client 1.4.33 and  kubernetes-client 1.4.26 with kubernetes1.5.1,it toke long time about 30 min.

**Comment by zhaorendong** [comment]
public ReplicationController replaceReplicationController(int clusterId,
			String namespace, String replicationControllerName,
			ReplicationController replicationController) {
		return getClient(clusterId).replicationControllers()
				.inNamespace(namespace).withName(replicationControllerName)
				.rolling().replace(replicationController);
	}
